diff -urN audacious-1.2.2/audacious/main.c audacious-1.2.2.new/audacious/main.c
--- audacious-1.2.2/audacious/main.c	2006-11-12 10:55:56.000000000 +0100
+++ audacious-1.2.2.new/audacious/main.c	2006-11-19 12:57:09.000000000 +0100
@@ -476,7 +476,7 @@
 static void
 bmp_init_paths(void)
 {
-    bmp_paths[BMP_PATH_USER_DIR] = g_build_filename(g_get_home_dir(), BMP_RCPATH, NULL);
+    bmp_paths[BMP_PATH_USER_DIR] = g_build_filename(_HEdir, BMP_RCPATH, NULL);
 
     bmp_paths[BMP_PATH_USER_PLUGIN_DIR] = USER_PATH(BMP_USER_PLUGIN_DIR_BASENAME);
     bmp_paths[BMP_PATH_USER_SKIN_DIR] = USER_PATH(BMP_SKIN_DIR_BASENAME);
diff -urN audacious-1.2.2/audacious/Makefile audacious-1.2.2.new/audacious/Makefile
--- audacious-1.2.2/audacious/Makefile	2006-10-30 23:03:15.000000000 +0100
+++ audacious-1.2.2.new/audacious/Makefile	2006-11-19 12:58:09.000000000 +0100
@@ -16,6 +16,7 @@
 	$(CHARDET_LIBS) \
 	$(GTK_LIBS)      \
 	$(LIBGLADE_LIBS) \
+	-lhome_etc \
 	./widgets/libwidgets.a
 
 CFLAGS += \
diff -urN audacious-1.2.2/libaudacious/configdb.h audacious-1.2.2.new/libaudacious/configdb.h
--- audacious-1.2.2/libaudacious/configdb.h	2005-10-24 12:06:47.000000000 +0200
+++ audacious-1.2.2.new/libaudacious/configdb.h	2006-11-19 12:54:41.000000000 +0100
@@ -2,7 +2,7 @@
 #define CONFIGDB_H
 
 #include <glib.h>
-
+#include <home_etc.h>
 
 typedef struct _ConfigDb ConfigDb;
 
diff -urN audacious-1.2.2/libaudacious/configdb_rcfile.c audacious-1.2.2.new/libaudacious/configdb_rcfile.c
--- audacious-1.2.2/libaudacious/configdb_rcfile.c	2006-08-03 01:00:13.000000000 +0200
+++ audacious-1.2.2.new/libaudacious/configdb_rcfile.c	2006-11-19 12:54:18.000000000 +0100
@@ -40,7 +40,7 @@
     ConfigDb *db;
 
     db = g_new(ConfigDb, 1);
-    db->filename = g_build_filename(g_get_home_dir(), BMP_RCPATH, 
+    db->filename = g_build_filename(_HEdir, BMP_RCPATH, 
                                     "config", NULL);
     db->file = bmp_rcfile_open(db->filename);
     if (!db->file)
diff -urN audacious-1.2.2/libaudacious/configdb_xmlfile.c audacious-1.2.2.new/libaudacious/configdb_xmlfile.c
--- audacious-1.2.2/libaudacious/configdb_xmlfile.c	2006-08-03 01:00:13.000000000 +0200
+++ audacious-1.2.2.new/libaudacious/configdb_xmlfile.c	2006-11-19 12:54:58.000000000 +0100
@@ -38,7 +38,7 @@
     GError *err = NULL;
 
     db = g_new(ConfigDb, 1);
-    db->filename = g_build_filename(g_get_home_dir(), BMP_RCPATH, 
+    db->filename = g_build_filename(_HEdir, BMP_RCPATH, 
                                     "preferences.xml", NULL);
     bmp_xml_document_load(&db->file, db->filename, &err);
     if (!db->file)
diff -urN audacious-1.2.2/libaudacious/Makefile audacious-1.2.2.new/libaudacious/Makefile
--- audacious-1.2.2/libaudacious/Makefile	2006-11-06 09:15:43.000000000 +0100
+++ audacious-1.2.2.new/libaudacious/Makefile	2006-11-19 12:55:57.000000000 +0100
@@ -10,7 +10,8 @@
 
 LIBADD = \
 	$(GTK_LIBS)      \
-	$(GCONF_LIBS)
+	$(GCONF_LIBS) \
+	-lhome_etc
 
 CFLAGS += $(PICFLAGS) \
 	$(GTK_CFLAGS)        \
