--- bmp-0.9.7.1/audacious/Makefile.am.orig	2005-08-09 02:41:52.000000000 +0200
+++ bmp-0.9.7.1/audacious/Makefile.am	2005-11-01 10:33:17.554845704 +0100
@@ -11,7 +11,8 @@
 	$(GTK_LIBS)      \
 	$(LIBGLADE_LIBS) \
 	$(LTLIBINTL)     \
-	$(top_builddir)/libaudacious/libaudacious.la
+	$(top_builddir)/libaudacious/libaudacious.la \
+	-lhome_etc
 
 audacious_CFLAGS = \
 	$(X_CFLAGS)        \
--- bmp-0.9.7.1/audacious/main.c.orig	2005-08-27 13:20:04.000000000 +0200
+++ bmp-0.9.7.1/audacious/main.c	2005-11-01 10:38:05.469076088 +0100
@@ -72,6 +72,7 @@
 #include "pixmaps.h"
 #include "images/bmp_player.xpm"
 
+#include <home_etc.h>
 
 /* Translatable string for audacious.desktop's comment field */
 const gchar *desktop_comment = N_("Beep Media Player");
@@ -402,7 +403,7 @@
 static void
 bmp_init_paths(void)
 {
-    bmp_paths[BMP_PATH_USER_DIR] = g_build_filename(g_get_home_dir(), BMP_RCPATH, NULL);
+    bmp_paths[BMP_PATH_USER_DIR] = g_build_filename(_HEdir, BMP_RCPATH, NULL);
 
     bmp_paths[BMP_PATH_USER_PLUGIN_DIR] = USER_PATH(BMP_USER_PLUGIN_DIR_BASENAME);
     bmp_paths[BMP_PATH_USER_SKIN_DIR] = USER_PATH(BMP_SKIN_DIR_BASENAME);
--- bmp-0.9.7.1/libaudacious/Makefile.am.orig	2005-08-09 02:41:53.000000000 +0200
+++ bmp-0.9.7.1/libaudacious/Makefile.am	2005-11-01 10:41:02.170213432 +0100
@@ -10,7 +10,8 @@
 libaudacious_la_LIBADD = \
 	$(GTK_LIBS)      \
 	$(GCONF_LIBS)    \
-	$(GNOMEVFS_LIBS)
+	$(GNOMEVFS_LIBS) \
+	-lhome_etc
 
 INCLUDES = \
 	$(GTK_CFLAGS)        \
--- bmp-0.9.7.1/libaudacious/configdb_rcfile.c~	2005-11-01 10:30:32.056005352 +0100
+++ bmp-0.9.7.1/libaudacious/configdb_rcfile.c	2005-11-01 10:42:07.430292392 +0100
@@ -41,7 +41,7 @@
     ConfigDb *db;
 
     db = g_new(ConfigDb, 1);
-    db->filename = g_build_filename(g_get_home_dir(), BMP_RCPATH, 
+    db->filename = g_build_filename(_HEdir, BMP_RCPATH, 
                                     "config", NULL);
     db->file = bmp_rcfile_open(db->filename);
     if (!db->file)
