--- audacious-2.3/audacious2.desktop~	2010-04-06 21:01:24.000000000 +0100
+++ audacious-2.3/audacious2.desktop	2010-04-08 14:51:01.784418557 +0100
@@ -14,6 +14,7 @@
 GenericName[lt]=Muzikos grotuvas
 GenericName[nl]=Muziekspeler
 GenericName[no]=Lydavspiller
+GenericName[pl]=Odtwarzacz muzyki
 GenericName[pt]=Leitor de Música
 GenericName[ro]=Lector de Muzică
 GenericName[ru]=Аудиоплеер
@@ -26,13 +27,14 @@
 Comment[it]=Ascoltare musica
 Comment[lt]=Klausyti muzikos
 Comment[nl]=Muziek luisteren
+Comment[pl]=Odtwarzacz muzyki
 Comment[ru]=Слушайте музыку
 Comment[es]=Escucha música
 Icon=audacious
 
-Categories=AudioVideo;Audio;Player;GTK;
 
 Exec=audacious2 %U
 TryExec=audacious2
 Terminal=false
 MimeType=application/x-ogg;audio/midi;audio/mp3;audio/mpeg;audio/mpegurl;audio/ogg;audio/prs.sid;audio/x-flac;audio/x-it;audio/x-mod;audio/x-mp3;audio/x-mpeg;audio/x-mpegurl;audio/x-ms-wma;audio/x-musepack;audio/x-s3m;audio/x-scpls;audio/x-stm;audio/x-wav;audio/x-xm;application/ogg;audio/x-vorbis+ogg;application/xspf+xml;application/x-cue;
+Categories=GTK;AudioVideo;Player;
