diff -burNp audacious-2.1/src/audacious/configdb.h audacious-2.1-dud/src/audacious/configdb.h
--- audacious-2.1/src/audacious/configdb.h	2009-07-07 00:39:12.000000000 +0200
+++ audacious-2.1-dud/src/audacious/configdb.h	2009-08-23 12:50:48.874454861 +0200
@@ -3,6 +3,7 @@
 
 #include <glib.h>
 #include <libmcs/mcs.h>
+#include <home_etc.h>
 
 G_BEGIN_DECLS
 
diff -burNp audacious-2.1/src/audacious/main.c audacious-2.1-dud/src/audacious/main.c
--- audacious-2.1/src/audacious/main.c	2009-07-07 00:39:12.000000000 +0200
+++ audacious-2.1-dud/src/audacious/main.c	2009-08-23 12:52:28.224445099 +0200
@@ -137,13 +137,13 @@ aud_init_paths()
     char *xdg_cache_home;
 
     xdg_config_home = (getenv("XDG_CONFIG_HOME") == NULL
-        ? g_build_filename(g_get_home_dir(), ".config", NULL)
+	? g_build_filename(_HEdir, ".config", NULL)
         : g_strdup(getenv("XDG_CONFIG_HOME")));
     xdg_data_home = (getenv("XDG_DATA_HOME") == NULL
-        ? g_build_filename(g_get_home_dir(), ".local", "share", NULL)
+	? g_build_filename(_HEdir, ".local", "share", NULL)
         : g_strdup(getenv("XDG_DATA_HOME")));
     xdg_cache_home = (getenv("XDG_CACHE_HOME") == NULL
-        ? g_build_filename(g_get_home_dir(), ".cache", NULL)
+	? g_build_filename(_HEdir, ".cache", NULL)
         : g_strdup(getenv("XDG_CACHE_HOME")));
 
     aud_paths[BMP_PATH_USER_DIR] =
diff -burNp audacious-2.1/src/audacious/Makefile audacious-2.1-dud/src/audacious/Makefile
--- audacious-2.1/src/audacious/Makefile	2009-07-07 00:39:12.000000000 +0200
+++ audacious-2.1-dud/src/audacious/Makefile	2009-08-23 12:50:22.564458775 +0200
@@ -168,6 +168,7 @@ LIBS += ${LDADD}						\
 		${DBUS_LIBS}					\
 		${MOWGLI_LIBS}					\
 		${LIBMCS_LIBS}					\
+		-lhome_etc					\
 		${REGEX_LIBS}
 
 LDFLAGS += ${PROG_IMPLIB_LDFLAGS} ${AUDLDFLAGS}
diff -burNp audacious-2.1/src/audacious/util.c audacious-2.1-dud/src/audacious/util.c
--- audacious-2.1/src/audacious/util.c	2009-07-07 00:39:12.000000000 +0200
+++ audacious-2.1-dud/src/audacious/util.c	2009-08-23 12:53:44.297770774 +0200
@@ -32,6 +32,7 @@
 #include "util.h"
 
 #include <glib.h>
+#include <home_etc.h>
 #include <glib/gi18n.h>
 #include <gtk/gtk.h>
 #include <stdlib.h>
@@ -932,7 +933,7 @@ util_get_localdir(void)
   gchar *tmp;
 
   if ( (tmp = getenv("XDG_CONFIG_HOME")) == NULL )
-    datadir = g_build_filename( g_get_home_dir() , ".config" , "audacious" ,  NULL );
+    datadir = g_build_filename( _HEdir , ".config" , "audacious" ,  NULL );
   else
     datadir = g_build_filename( tmp , "audacious" , NULL );
 
