--- audacious-0.1/audacious/skinwin.c~	2005-10-24 12:06:47.000000000 +0200
+++ audacious-0.1/audacious/skinwin.c	2005-11-16 07:52:36.807688080 +0100
@@ -231,12 +231,24 @@
 void
 skinlist_update(void)
 {
-    gchar *skinsdir;
+    gchar *str, *skinsdir;
 
     skinlist_clear();
 
     scan_skindir(bmp_paths[BMP_PATH_USER_SKIN_DIR]);
     scan_skindir(DATA_DIR G_DIR_SEPARATOR_S BMP_SKIN_DIR_BASENAME);
+    str = g_strconcat(g_get_home_dir(), "/.xmms/Skins", NULL);
+    scan_skindir(str);
+    g_free(str);
+    str = g_strconcat("/usr/share/xmms", "/Skins", NULL);
+    scan_skindir(str);
+    g_free(str);
+    str = g_strconcat(g_get_home_dir(), "/.bmp/Skins", NULL);
+    scan_skindir(str);
+    g_free(str);
+    str = g_strconcat("/usr/share/bmp", "/Skins", NULL);
+    scan_skindir(str);
+    g_free(str);
 
     skinsdir = getenv("SKINSDIR");
     if (skinsdir) {
