diff -aurN audacious-0.2.orig/audacious/audacious.desktop.in audacious-0.2/audacious/audacious.desktop.in
--- audacious-0.2.orig/audacious/audacious.desktop.in	2006-01-03 22:05:01.000000000 +0100
+++ audacious-0.2/audacious/audacious.desktop.in	2006-02-09 17:07:53.848956736 +0100
@@ -2,10 +2,11 @@
 Name=Audacious
 Comment=Play music
 Comment[hu]=Zene lejátszása
+Comment[pl]=Odtwarzacz muzyki
 Exec=audacious
-Icon=audacious.png
+Icon=audacious
 MimeType=audio/x-scpls;audio/x-mpegurl;audio/mpegurl;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;application/x-ogg;
-Categories=Application;GTK;AudioVideo;Player;
+Categories=GTK;Audio;Player;
 Terminal=false
 Type=Application
 Encoding=UTF-8
